dist/images/ovnkube.sh

```diff
@@ -1251,6 +1340,8 @@ ovn-network-controller-manager() {
     ${egressservice_enabled_flag} \
     ${ovnkube_config_duration_enable_flag} \
     ${multi_network_enabled_flag} \
+    ${ovnkube_enable_interconnect_flag} \
+    --zone ${ovn_zone} \
```

Comment:
If we don't pass,  then until ovnkube-controller sees the zone value in the db, it will assume its in global zone.

The rationale for adding this config is mentioned in this commit - https://github.com/ovn-org/ovn-kubernetes/commit/f7c40b0006fd7e2ef0bbb51bef35029aa9894a2d

Two main reasons
1.  Yes for upgrade scenarios.
2.  Until the OVN dbs and ovn-northd comes up and copies the NB zone to the sB Zone, ovnkube-node will set its zone as "global".   This will cause the ovnkube-controller in the global zone to consider this node as part of its zone.

Reply:
oh I see. Thanks for explaining.